pages/osx/valet.md

```diff
@@ -0,0 +1,24 @@
+# valet
+
+> A minimalist Laravel development environment that allows hosting sites via local tunnels on `http://<example>.test`.
+> More information: <https://laravel.com/docs/5.8/valet>.
+
+- Start the valet daemon:
+
+`valet start`
+
+- Register your current working directory as a path that Valet should search for sites:
+
+`valet park`
+
+- View your 'parked' paths:
```

Comment:
We can just remove "your" here. The meaning remains the same.

Reply:
done